src/core/DSPCoreValueSets.ttl

```diff
@@ -1,9 +1,9 @@
-# baseURI: http://datamodel.broadinstitute.org/DSPCoreValueSets
+# baseURI: http://datamodel.terra.bio/DSPCoreValueSets
 # imports: http://datamodel.terra.bio/UBERON_subset.owl
 # imports: http://www.w3.org/2004/02/skos/core#
-# prefix: DSPCoreValueSets
 
-@prefix DSPCoreValueSets: <http://datamodel.broadinstitute.org/DSPCoreValueSets#> .
+@prefix : <http://datamodel.terra.bio/DSPCoreValueSets#> .
```

Comment:
Is this an example of a "default namespace prefix"? As opposed to the following line which is an example of a namespace prefix assigned to a string? Just curious as I was trying to read up on Turtle file syntax to learn some more about these files.